disco/extensions/upgrade_simulation/upgrade_simulation.py

```diff
@@ -49,6 +49,9 @@ def get_thermal_upgrades_dss_file(self):
     def get_voltage_upgrades_dss_file(self):
         voltage_upgrades = self.get_voltage_upgrades_directory()
         return os.path.join(voltage_upgrades, "voltage_upgrades.dss")
+    
+    def get_upgraded_master_dss_file(self):
```

Comment:
Would you mind adding a test that compiles the upgraded master file in OpenDSS to make sure that it's functional?

Reply:
The upgraded master.dss is called in both the thermal and voltage upgrades scripts in the end. Is that fine? It will raise error if it doesnt compile. 
Or is it good to still create one in the test_generic_upgrade.py file?

Reply:
Ok, that works for me.